doc/content/reference/interop-repository/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Interoperability Repository"
+description: ""
+weight: 3
+---
+
+## Interoperability Repository
+
+The stack expects the root of interoperability repository to contain `config.yaml`, which contains various interoperablity configuration options:
+
+```yaml
+join-servers:                 # list of Join Server interoperability configurations,
+                              # used to map a JoinEUI to the Join Server
+  - file: './path/js.yaml'    # relative path to a file containing Join Server configiration
+    join-euis:                # list of Join EUI prefixes the Join Server should handle
+    - '11aa000000000000/16'
+```
+
+- All paths are relative to the file they're defined in, that is `example/js.yaml` defined in `interopconf/config.yaml` is expected to be located at `interopconf/example/js.yaml`.
+- For `join-servers`, in case Join EUI prefixes overlap, the most specific prefix takes precedence.
+- Interoperability Join Server requests are only performed if either:
+  a. Cluster-local Join Server is not available (e.g. it's disabled in the stack or it's down)
+  b. Cluster-local Join Server is available, but does not contain the information about a device indentified by the given JoinEUI and DevEUI.
+- `0000000000000000/0` prefix matches all JoinEUIs, while being the least specific, hence it always matches last. Use this prefix if you wish to define a "fallback" Join Server interoperability configuration.
```

Comment:
- Prefer text over bullets
- Write `JoinEUI` (literal), not "Join EUI", "JoinEUI"
- No shortening; "it is" over "it's" and "they are" over "they're"
- `yml` over `yaml`